src/main/java/kitchenpos/menu/exception/MenuException.java

```diff
@@ -0,0 +1,7 @@
+package kitchenpos.menu.exception;
+
+public class MenuException extends RuntimeException {
```

Comment:
상황에 맞는 `Exception` 너무 좋습니다 😄 
이렇게 `Customized`된 `Exception`이라면 기본 `DEFAULT_MESSAGE`를 가지고 있어도 좋을 것 같습니다 😄 
```suggestion
private static final String DEFAULT_MESSAGE = "메뉴를 찾을 수 없습니다";

public MenuException() {
    super(DEFAULT_MESSAGE);
}
```

그럼 왠만한 기본 예외에는 따로 메세지를 전달하는 수고(?)를 줄일 수 있을 것 같습니다 😄 
이 부분도 참고만 부탁 드립니다 😄 🙇